kubernetes/gitlab-auth.sh

```diff
@@ -34,117 +34,116 @@ runCmd() {
   (set -x; $@)
 }
 
-# getGitlabToken generate Gitlab personal access token with api scopes
+# genGitlabToken generates Gitlab personal access token with 'api' scope.
+#
 # Example:
-#   getGitlabToken "https://gitlab.com" "user" "pass" "token name"
-generateGitlabToken() {
-  local GITLAB_URL=$1
-  local GITLAB_USER=$2
-  local GITLAB_PASS=$3
-  local GITLAB_TOKEN_NAME=$4
-  local COOKIES_FILE=$(mktemp --suffix=cookies)
+#   genGitlabToken "https://gitlab.com" "user" "pass" "token-name"
+genGitlabToken() {
+  local GITLAB_URL="$1"
+  local GITLAB_USER="$2"
+  local GITLAB_PASS="$3"
+  local GITLAB_TOKEN_NAME="$4"
+
+  local COOKIES_FILE="$(mktemp --suffix=cookies)"
 
+  # 1. Sign in into GitLab via username and password.
   local htmlContent=$(curl -c "$COOKIES_FILE" -i "$GITLAB_URL/users/sign_in" -s)
-  local csrfToken=$(echo $htmlContent \
+  local csrfToken=$(echo "$htmlContent" \
```

Comment:
@tyranron here and same places, quotes has broken functionality. I think cause of this is escapes symbols